frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/date_cell/date_cal_bloc.dart

```diff
@@ -121,6 +125,42 @@ class DateCellCalendarBloc
     );
   }
 
+  Future<void> _clearDate(Emitter<DateCellCalendarState> emit) async {
+    final DateCellData newDateData = DateCellData(
+      dateTime: null,
+      time: null,
+      includeTime: state.includeTime,
+      clearFlag: true,
+    );
+
+    cellController.saveCellData(
+      newDateData,
+      onFinish: (result) {
+        result.fold(
+          () {
+            if (!isClosed && state.timeFormatError != null) {
+              add(const DateCellCalendarEvent.didReceiveTimeFormatError(null));
+            }
+          },
+          (err) {
+            switch (ErrorCode.valueOf(err.code)!) {
+              case ErrorCode.InvalidDateTimeFormat:
+                if (isClosed) return;
+                add(
+                  DateCellCalendarEvent.didReceiveTimeFormatError(
+                    timeFormatPrompt(err),
+                  ),
+                );
+                break;
```

Comment:
It should be okay to simply log the error if there is one. I don't think that time format error can occur when clearing the date.